### DISCUSSION TITLE
How to retrieve value from 0d array? (obtained with `max()`)

### DISCUSSION THREAD

**Discussion by yohplala** [vaexio/vaex#1699] [Q&A] [answered]
Hi,
This is probably a very newbie question, but `max()`  returns a 0d numpy array.
I would like to get the result as an `int` (here), but I am unable to by using cell index.

```python
import vaex as vx
li = [1, 199, 79]
vdf = vx.from_arrays(id = li)
max_id = vdf['id'].max()[0]

Traceback (most recent call last):

  File "<ipython-input-15-2c73f0975c81>", line 4, in <module>
    max_id = vdf['id'].max()[0]

IndexError: too many indices for array: array is 0-dimensional, but 1 were indexed
```

Please, what is the 'right' way of doing?

I can see it works by 'forcing' the array to `int`, but this approach appears to me a bit weird somehow.

```python
max_id = int(vdf['id'].max())
```

Any advice is welcome! :)
Bests

**Comment by JovanVeljanoski** [comment, marked as answer]
Yeah, this is just `numpy` stuff. You can do what you want with `.item()`. Like this

```python
df["my_column"].max().item()
```

**Reply by yohplala** [reply]
Thanks, this does the trick!
